core/src/crds_value.rs

```diff
@@ -61,7 +59,7 @@ impl Signable for CrdsValue {
 pub enum CrdsData {
     ContactInfo(ContactInfo),
     Vote(VoteIndex, Vote),
-    EpochSlots(EpochSlotIndex, EpochSlots),
+    LowestSlot(LowestSlot),
```

Comment:
But you said no more ABI changes last month!

This is fine I guess, we'll just be unable to do a rolling update from 1.0.x to 1.1.x next month.

Reply:
Rolling update should still work.  Repair will be slow until the vast majority updates.  We don’t store the crds on disk.

Reply:
@carllin wdyt?  should i keep the old fields in there and mark them as deprecated?  In theory the upgrade should allow both to exist, you just don't want to be in the minority.

Reply:
the annoying thing is that it pulls in a big pile of crap, EpochIncompleteSlots, and all the compression stuff